app.py

```diff
@@ -80,7 +80,8 @@ def vote(vote_button, response_a, response_b, model_a_name, model_b_name,
 """
 
 with gr.Blocks(title="Yanolja Arena", css=css) as app:
-  set_token(app)
+  token = gr.Textbox(visible=False)
```

Comment:
what's the reason why token is a text box? like, not just string?
https://github.com/yanolja/arena/pull/123/files#diff-b2d7b04ca1a7dab749196b5980281f02920d9a1948533731f5b1a5cbd7ac5999R102
it's a plain string again so I'm confused a bit.

Reply:
string variables cannot be used as Gradio inputs and outputs, so I used Textbox for storing a string value.
gr.State could be an option but currently there is a bug with JS